R/efp.R

```diff
@@ -7,21 +7,45 @@ efp <- function(X, ...)
 efp.formula <- function(formula, data = list(),
                 type = c("Rec-CUSUM", "OLS-CUSUM", "Rec-MOSUM", "OLS-MOSUM",
                 "RE", "ME", "Score-CUSUM", "Score-MOSUM", "fluctuation"),
-                h = 0.15, dynamic = FALSE, rescale = TRUE, ...)
+                h = 0.15, dynamic = FALSE, rescale = TRUE, lrvar = FALSE, vcov = NULL, ...)
```

Comment:
@appelmar Could you check this part? Achim implemented two optional parameters to efp[.formula], could this be ported to the matrix function as well? Or maybe we should merge the two functions so that changes like that are easier to integrate?